src/controller/WebController.php

```diff
@@ -341,7 +324,8 @@ public function sendFeedback($request, $message, $messageSubject, $fromName = nu
     public function invokeAboutPage($request)
     {
         $template = $this->twig->loadTemplate('about.twig');
-        $this->setLanguageProperties($request->getLang());
+        //$this->setLanguageProperties($request->getLang());
```

Comment:
same as above, remove

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764